test/domUtil.test.js

```diff
@@ -177,6 +177,24 @@ describe('DomUtil', function() {
         assert.strictEqual(fromJSON({ "a": undefined }), '<root/>');
     });
 
+    describe('fromJSON (advanced)', function() {
+        function fromJSON(json) {
+            var xml = DomUtil.fromJSON("root", json);
+            return DomUtil.toXMLString(xml);
+        }
+
+        it("Should handle fixes made in version 1.1.3", () => {
+            expect(fromJSON({ $: "Hello" })).toBe("<root>Hello</root>");
+            expect(fromJSON({ "$delivery": "Hello" })).toBe("<root><delivery>Hello</delivery></root>");
+            expect(fromJSON({ "delivery": { $: "Hello" } })).toBe("<root><delivery>Hello</delivery></root>");
+            //expect(fromJSON({ "$delivery": "World", "delivery": { $: "Hello" } })).toBe("<root><delivery>Hello</delivery></root>");
```

Comment:
Debug code? Perhaps, your intention is to fix this case in the future. We could leave it like that then.
The asserted behaviour seems correct to me.